index.bs

```diff
@@ -234,28 +237,25 @@ class AV1CodecConfigurationBox extends FullBox('av1C', version = 0, 0){
 
 aligned (8) class AV1CodecConfigurationRecordv1 {
   unsigned int (1) marker = 1;
-  unsigned int (3) version = 1;
+  unsigned int (7) version = 1;
```

Comment:
This could also be reserved (4) version (3). I don't think there will ever be a need to add any other field to the "header" byte, but there are certainly more chances of that than of version ever needing seven bits :p

Keep in mind the size of the version field can't change once it's frozen, as it would require bumping the av1C fullbox version to define a new codec configuration record struct.